src/sysext/sysext.c

```diff
@@ -1746,6 +1746,9 @@ static int merge(ImageClass image_class,
         if (r == 123) /* exit code 123 means: didn't do anything */
                 return 0;
 
+        if (r > 0)
```

Comment:
Please remove the empty lines above and the one above `r == 123` too.

Reply:
Done.